xamarin/realm-tutorial-dotnet/App.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Xamarin.Forms;
+using System.IO;
+using System.Xml;
+using System;
+
+namespace RealmTemplateApp
+{
+    public partial class App : Application
+    {
+        private string appId;
+        public static Realms.Sync.App RealmApp;
+
+        public App()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnStart()
+        {
+            try
+            {
+                appId = GetAppId();
+                RealmApp = Realms.Sync.App.Create(appId);
+
+                var navPage = App.RealmApp.CurrentUser == null ?
+                    new NavigationPage(new LoginPage()) :
+                    new NavigationPage(new TaskPage());
+
+                NavigationPage.SetHasBackButton(navPage, false);
+                MainPage = navPage;
+            }
+            catch (NullReferenceException nre)
```

Comment:
hahaha. I added it because once a user starts using the app, they are likely to make a mistake in the file (like remove it).Keeping the `catch` in there with a note will hopefully be enough for them to see what's likely wrong, and sine this happens before a view is loaded, I can't just display a toast. Other thoughts?

Reply:
Wouldn't swallowing the error make it more likely that the user would miss this message?